tools/Import/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Import
+{
+    public static class Program
```

Comment:
While optional, the AAD authentication module using client id and client secret can add value to the tool.

Reply:
I agree. It can be added later in different PR.